docs/hub/organizations-security.md

```diff
@@ -38,10 +38,10 @@ As an organization `admin`, go to the **Members** section of the org settings to
 This feature is part of the <a href="https://huggingface.co/enterprise" target="_blank">Enterprise Hub</a>.
 </Tip>
 
-You may be able to view the email addresses for members of your organization. The visibility of the email addresses depends on the organization's sso configuration, or verified organization status.
+You may be able to view the email addresses of members of your organization. The visibility of the email addresses depends on the organization's SSO configuration, or verified organization status.
 
-- If you verify a domain for your organization, you can view members' email addresses for the verified domain. [Organization domain name](./organizations-managing#organization-domain-name)
-- If SSO is configured for your organization, you can view the email address for each of your organization members by setting `Matching email domains` in SSO configuration  
+- If you [verify a domain for your organization](./organizations-managing#organization-domain-name), you can view members' email addresses for the verified domain.
```

Comment:
note that the link links to something that's too close to the bottom of the page, so it's not very clear what it links too (IMO we should highlight the linked part in yellow – some websites do that – maybe cc @mishig25 for viz on this frontend feature)